gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/EntitiesRequestContext.java

```diff
@@ -19,4 +20,18 @@ public EntitiesRequestContext(
   public String getEntityType() {
     return this.entityType;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Is this used in any maps?

Reply:
I'm checking for equality in unit tests.